starter-code/lib/harryPotter.js

```diff
@@ -1,10 +1,21 @@
- var birthdays = ["Severus Snape", "9 January", "Arthur Weasley", "6 February",
-                  "Ron Weasley", "1 March", "Remus Lupin", "10 March",
-                  "Fred & George Weasley", "1 April", "Pomona Sprout", "15 May",
-                   "Draco Malfoy", "5 June", "Dobby", "28 June",
-                   "Neville Longbottom", "30 July", "Harry Potter", "31 July",
-                   "Ginevra Weasley", "11 August", "Percy Weasley ", "22 August",
-                   "Hermione Granger", "19 September", "Minerva McGonagall",
-                   "4 October", "Filius Flitwick", "17 October", "Molly Weasley",
-                   "30 October", "Bill Weasley", "29 November", "Rubeus Hagrid",
-                   "6 December", "Charlie Weasley", "12 December"];
+//var _ = require("lodash");
+
+var birthdays = ["Severus Snape", "9 January", "Arthur Weasley", "6 February",
+                 "Ron Weasley", "1 March", "Remus Lupin", "10 March",
+                 "Fred & George Weasley", "1 April", "Pomona Sprout", "15 May",
+                  "Draco Malfoy", "5 June", "Dobby", "28 June",
+                  "Neville Longbottom", "30 July", "Harry Potter", "31 July",
+                  "Ginevra Weasley", "11 August", "Percy Weasley ", "22 August",
+                  "Hermione Granger", "19 September", "Minerva McGonagall",
+                  "4 October", "Filius Flitwick", "17 October", "Molly Weasley",
+                  "30 October", "Bill Weasley", "29 November", "Rubeus Hagrid",
+                  "6 December", "Charlie Weasley", "12 December"];
+
+var temp = [];
+var newArray = [];
+newArray = _.times((birthdays.length/2), function () {
+ temp = _.pullAt(birthdays, [0, 1]);
+ return temp;
+});
+
```

Comment:
You could've used the chunk method:

var pairsOfBirthdays = _.chunk(birthdays, 2);

and then the concat method to add the other birthdays:

var allBirthdays = function () { return _.concat(pairsOfBirthdays, _.chunk(moreBirthdays, 2));};